sparkle/src/components/markdown/CodeBlock.tsx

```diff
@@ -139,7 +139,7 @@ export function CodeBlock({
       className={cn(
         "s-mx-0.5 s-cursor-text s-rounded-lg s-border s-px-1.5 s-py-1",
         "s-border-border-dark dark:s-border-border-dark-night",
-        "dark:s-bg-muted-night s-bg-muted",
+        "s-bg-muted dark:s-bg-muted-night",
```

Comment:
I ran the linter so in a couple of places like this it's just an ordering diff, no change on the classes just their order